### ISSUE TITLE
Missing focus indicator if user tabs to element

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/tiamat-theme#1665]
From the digital accessibility team:

As a part of a test, we were reviewing the [Academic Leaders Institute page](https://www.colorado.edu/facultyaffairs/academic-leaders-institute). The expandable element (See a list of our yearly workshop topics) does not get a focus indicator if a user tabs to it. That’s the only actionable element that is missing the focus indicator, so we wanted to point it out.

https://www.colorado.edu/facultyaffairs/academic-leaders-institute
